client/devdocs/design/component-examples.js

```diff
@@ -1,5 +1,6 @@
 export { default as ActionCard } from 'calypso/components/action-card/docs/example';
 export { default as Animate } from 'calypso/components/animate/docs/example';
+export { default as AppPromo } from 'calypso/components/app-promo/docs/example';
```

Comment:
This needs to be removed now that we've changed the component to a block.

```suggestion
```

Reply:
Resolved it in https://github.com/Automattic/wp-calypso/pull/83014/commits/7b53502841623fefb22b6488746a7734fc8deab5